pandas/io/formats/style.py

```diff
@@ -1148,9 +1149,9 @@ def _apply(
         subset = slice(None) if subset is None else subset
         subset = non_reducing_slice(subset)
         data = self.data.loc[subset]
-        if axis in [0, "index"]:
-            result = data.apply(func, axis=0, **kwargs)
-        elif axis in [1, "columns"]:
+        if DataFrame()._get_axis_number(axis)==0:
+         result = data.apply(func, axis=0, **kwargs)
+        if DataFrame()._get_axis_number(axis)==1:
```

Comment:
why?

Reply:
don't make changes to `style.py` this already has PRs fixing this. please revert.

Reply:
aight. on it